File: base64sample.rb
require "base64"

s1 = 'SXQncyBsdW5jaCB0aW1l'
s_decoded = Base64.decode64(s1)
puts s_decoded

s2 = 'time for lunch'
s_encoded = Base64.encode64(s2)
puts s_encoded

File: base64sample.pl
use MIME::Base64;
use feature 'say';

$encoded = 'SXQncyBsdW5jaCB0aW1l';
$decoded = decode_base64($encoded);
say $decoded;

$string = 'time for lunch';
$encoded = encode_base64($string);
say $encoded
